attendance/templates/attendance_base.html

```diff
@@ -0,0 +1,209 @@
+<!DOCTYPE html>
+{% load static %}
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <link href="//cdn.muicss.com/mui-0.9.43/css/mui.min.css" rel="stylesheet" type="text/css" />
+    <link href='https://fonts.googleapis.com/css?family=Montserrat' rel='stylesheet'>
+    <link rel="stylesheet" href="{% static  'css/mui_assets.css' %}"/>
+    <link rel="icon" href="https://github.com/Ernesto2004/MB_Portal_Design/blob/master/mblogo.png?raw=true"> <!-- LOGO ON THE TAB BAR -->
```

Comment:
The logo should be hosted from this project, e.g. using `{% static … %}`

Reply:
Agreed, will address

Reply:
Resolved in c23016220db3bfc353d37231de915ab5662abd51 and 01b1b470c564370935c87ea4d0ff50cef528011c